src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -219,7 +219,7 @@ def init_worker(self):
                     self._process_list.append(
                         Popen(
                             args,
-                            shell=constants.IS_WIN,
```

Comment:
Have you tested the change on Windows?

Reply:
Sure, https://github.com/openvinotoolkit/openvino/actions/runs/7119850501/job/19385845842?pr=21502